swift/Sources/FlatBuffers/ByteBuffer.swift

```diff
@@ -235,6 +235,23 @@ public struct ByteBuffer {
     }
   }
 
+  /// Adds a `ContiguousBytes` to buffer memory
+  /// - Parameter value: bytes to copy
+  #if swift(>=5.0) && !os(WASI)
+  @inline(__always)
+  @usableFromInline
+  mutating func push(bytes: ContiguousBytes) {
+    bytes.withUnsafeBytes { ptr in
```

Comment:
@wtholliday through some testing at work it seems that using `withUnsafeBytes` is a bit slower than `withUnsafeMutableBytes`